shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -70,7 +70,7 @@ void Reset() {
 /**
  * Private interface declaration for FLEViewController.
  */
-@interface FLEViewController ()
+@interface FLEViewController () <FlutterBinaryMessenger>
```

Comment:
These ones have engines too right? Can we add the relaying mechanism here too since we don't want the desktop view to power the channels either (could be in a different PR).

Reply:
(discussed offline) Yep, we should do the same thing for desktop.  I refrained from doing it now because the way the build system is setup it isn't easy to share the code currently.  This should present no change to the current functionality of the code while putting in place the API we want.